src/main/java/com/exasol/projectkeeper/ValidationFinding.java

```diff
@@ -69,4 +59,44 @@ public boolean hasFix() {
          */
         void fixError(Log log);
     }
+
+    /**
+     * Get a {@link Builder} for {@link ValidationFinding}.
+     * 
+     * @param message finding message
+     * @return builder
+     */
+    public static Builder withMessage(final String message) {
+        return new Builder(message);
+    }
+
+    /**
+     * Builder for {@link ValidationFinding}.
+     */
+    public static class Builder {
+        private final String message;
+        private Fix fix;
+
+        public Builder(final String message) {
+            this.message = message;
+        }
+
+        /**
+         * Add a fix to the finding (optional).
+         * <p>
+         * You can only add one fix. IF you call this method multiple times the fix will be overwritten.
+         * </p>
+         * 
+         * @param fix function that fixes the finding
+         * @return self for fluent programming
+         */
+        public Builder andFix(final Fix fix) {
```

Comment:
Is it intended to be single fix?

Another option would be to take a list of fixes and apply them in order.

Reply:
yes. There is always exactly one  fix per finding.